deps/spidershim/src/rootstore.h

```diff
@@ -119,10 +119,10 @@ class RootStore {
   }
 
  private:
-  JS::Rooted<ValueVector> values;
-  JS::Rooted<ScriptVector> scripts;
+  JS::PersistentRooted<ValueVector> values;
+  JS::PersistentRooted<ScriptVector> scripts;
   std::vector<Script*> scriptObjects;
-  JS::Rooted<SymbolVector> symbols;
+  JS::PersistentRooted<SymbolVector> symbols;
```

Comment:
I'll need to dig up the exact error, but it was rooted values being deconstructed in the wrong order, since we can't use the RAII approach of creating an isolate.